zio-http-gen/src/main/scala/zio/http/gen/scala/CodeGen.scala

```diff
@@ -250,17 +282,17 @@ object CodeGen {
     s""".header($headerSelector)"""
   }
 
-  def renderQueryCode(queryCode: Code.QueryParamCode): String = queryCode match {
+  def renderQueryCode(queryCode: Code.QueryParamCode): (List[Code.Import], String) = queryCode match {
     case Code.QueryParamCode(name, queryType) =>
-      val tpe = queryType match {
-        case Code.CodecType.Boolean => "Boolean"
-        case Code.CodecType.Int     => "Int"
-        case Code.CodecType.Long    => "Long"
-        case Code.CodecType.String  => "String"
-        case Code.CodecType.UUID    => "UUID"
+      val (imports, tpe) = queryType match {
+        case Code.CodecType.Boolean => Nil                                 -> "Boolean"
```

Comment:
Since we don't return a map here, can you change it to (a,b) instead of a -> b

Reply:
Do you mean in this place, or everywhere?